src/platyPS/platyPS.psm1

```diff
@@ -2542,7 +2542,8 @@ function ConvertPsObjectsToMamlModel
                 'InformationVariable',
                 'OutVariable',
                 'OutBuffer',
-                'PipelineVariable'
+                'PipelineVariable',
+                'ProgressAction'
```

Comment:
This whole section could be replaced with: `[System.Management.Automation.PSCmdlet]::CommonParameters` that way we can avoid this problem in the future.  
In the other sections where it's written out in a string you could do something like: `[System.Management.Automation.PSCmdlet]::CommonParameters | foreach -Begin {$Res = ""} -Process {$Res+= "-$_, "} -End {$Res.Trim(', ')}`

Reply:
Agree that this is the best way to solve for all.